setup/sql/dbupdate_custom_renameme.php

```diff
@@ -0,0 +1,7 @@
+<#1>
+<?php
+	if($ilDB->tableExists('svy_qst_oblig'))
+		$ilDB->dropTable('svy_qst_oblig');
+	if($ilDB->tableExists('svy_qst_oblig_seq'))
+		$ilDB->dropTable('svy_qst_oblig_seq');
+?>
```

Comment:
1) Please use
if($ilDB->sequenceExists('svy_qst_oblig'))
{
	$ilDB->dropSequence('svy_qst_oblig');
}
if($ilDB->tableExists('svy_qst_oblig'))
{
	$ilDB->dropTable('svy_qst_oblig');
}

2) But even before we need (imo) a migration:
The problem is that currently (5.1/5.2) the entries in this table seem to control what is really mandatory. Not the entries in svy_question. After this step svy_question would be relevant. Since it is possible to have inconsitencies by changing the value in the question form (and not in the table), this means the behaviour would change for these questions (with inconsistent values, e.g. mandatory set in svy_question and not set in svy_qst_oblig would suddenly make the question mandatory). So I think we need a migration of all the values of svy_qst_oblig into svy_question.